### ISSUE TITLE
Consider merging Flixel Power Tools into code tree

### ISSUE THREAD

**Issue by greysondn** [FlixelCommunity/flixel#163]
[website](http://www.photonstorm.com/flixel-power-tools)
[github](https://github.com/photonstorm/Flixel-Power-Tools)

Looking over his license, it seems somewhat sound for at least partial inclusion and/or merging. IANAL.

Richard Davey has noted that he is no longer working on this - having moved on to HTML 5. He had originally intended to add an incremental release for a few small fixes and additions; I don't know if he ever did, but it's been more than a year now. I think we can safely call the work dead where he stands. ([source, read comments](http://www.photonstorm.com/archives/2450/flixel-power-tools-v1-9-released))

**Comment by SeiferTim** [comment]
I agree with this, although someone should probably talk to him about it, first.
Since the Power Tools add so many great features that I use all the time, it makes sense. There are a few parts that are missing features or just need to be tweaked before I think it would be ready to be added in.

**Comment by Dovyski** [comment]
I like the idea too and agree we should exchange some ideas with Richard first. About the merge, I don't think we should do it for all the features.

I've used Power Tools before and some of its features are great to complement already existing code (e.g. shooting management), but sometimes they get into the way as you become more seasoned in Flixel. I would merge only the "core" features (e.g. pixel perfect collision) into Flixel, all the rest could be split into small plugins.

**Comment by greysondn** [comment]
I should mention that we could hang other suggestions I've had - #159 comes to mind - on this one. I've got lots of ideas but my math skills are weak XD So it's this balance of things that make sense to include "in general" or "for platformers" and "where can I give something back?"

Never simple ~ I have always been more an ideas and abstract code man than an actual get-it-done guy. But enough rambling.
